From 75c9e9324188b46a6a4745401857465736ec7dc3 Mon Sep 17 00:00:00 2001
From: Dillon Dumesnil <ddumesnil@edx.org>
Date: Mon, 24 May 2021 09:08:48 -0700
Subject: [PATCH 0474/1021] refactor: Update the defaults for our .env files
 (#459)

Nulls can provide undesired behavior so we want to switch to empty strings
---
 .env             | 62 ++++++++++++++++++++++++------------------------
 .env.development |  2 +-
 .env.test        |  2 +-
 3 files changed, 33 insertions(+), 33 deletions(-)

diff --git a/.env b/.env
index 61d7547..ebb0a7e 100644
--- a/.env
+++ b/.env
@@ -1,33 +1,33 @@
 NODE_ENV='production'
-ACCESS_TOKEN_COOKIE_NAME=null
-BASE_URL=null
-CREDENTIALS_BASE_URL=null
-CSRF_TOKEN_API_PATH=null
-DISCOVERY_API_BASE_URL=null
-ENTERPRISE_LEARNER_PORTAL_HOSTNAME=null
-ECOMMERCE_BASE_URL=null
+ACCESS_TOKEN_COOKIE_NAME=''
+BASE_URL=''
+CREDENTIALS_BASE_URL=''
+CSRF_TOKEN_API_PATH=''
+DISCOVERY_API_BASE_URL=''
+ECOMMERCE_BASE_URL=''
+ENTERPRISE_LEARNER_PORTAL_HOSTNAME=''
 IGNORED_ERROR_REGEX=''
-INSIGHTS_BASE_URL=null
-LANGUAGE_PREFERENCE_COOKIE_NAME=null
-LMS_BASE_URL=null
-LOGIN_URL=null
-LOGOUT_URL=null
-LOGO_URL=null
-LOGO_TRADEMARK_URL=null
-LOGO_WHITE_URL=null
-FAVICON_URL=null
-MARKETING_SITE_BASE_URL=null
-ORDER_HISTORY_URL=null
-REFRESH_ACCESS_TOKEN_ENDPOINT=null
-SEARCH_CATALOG_URL=null
-SEGMENT_KEY=null
-SITE_NAME=null
-SOCIAL_UTM_MILESTONE_CAMPAIGN=null
-STUDIO_BASE_URL=null
-SUPPORT_URL=null
-SUPPORT_URL_CALCULATOR_MATH=null
-SUPPORT_URL_ID_VERIFICATION=null
-SUPPORT_URL_VERIFIED_CERTIFICATE=null
-TWITTER_HASHTAG=null
-TWITTER_URL=null
-USER_INFO_COOKIE_NAME=null
+INSIGHTS_BASE_URL=''
+LANGUAGE_PREFERENCE_COOKIE_NAME=''
+LMS_BASE_URL=''
+LOGIN_URL=''
+LOGOUT_URL=''
+LOGO_URL=''
+LOGO_TRADEMARK_URL=''
+LOGO_WHITE_URL=''
+FAVICON_URL=''
+MARKETING_SITE_BASE_URL=''
+ORDER_HISTORY_URL=''
+REFRESH_ACCESS_TOKEN_ENDPOINT=''
+SEARCH_CATALOG_URL=''
+SEGMENT_KEY=''
+SITE_NAME=''
+SOCIAL_UTM_MILESTONE_CAMPAIGN=''
+STUDIO_BASE_URL=''
+SUPPORT_URL=''
+SUPPORT_URL_CALCULATOR_MATH=''
+SUPPORT_URL_ID_VERIFICATION=''
+SUPPORT_URL_VERIFIED_CERTIFICATE=''
+TWITTER_HASHTAG=''
+TWITTER_URL=''
+USER_INFO_COOKIE_NAME=''
diff --git a/.env.development b/.env.development
index c04bdef..2610c8b 100644
--- a/.env.development
+++ b/.env.development
@@ -20,7 +20,7 @@ ORDER_HISTORY_URL='http://localhost:1996/orders'
 PORT=2000
 REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 SEARCH_CATALOG_URL='http://localhost:18000/courses'
-SEGMENT_KEY=null
+SEGMENT_KEY=''
 SITE_NAME='edX'
 SOCIAL_UTM_MILESTONE_CAMPAIGN='edxmilestone'
 STUDIO_BASE_URL='http://localhost:18010'
diff --git a/.env.test b/.env.test
index 8ee1cf1..a9297d8 100644
--- a/.env.test
+++ b/.env.test
@@ -20,7 +20,7 @@ ORDER_HISTORY_URL='http://localhost:1996/orders'
 PORT=2000
 REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 SEARCH_CATALOG_URL='http://localhost:18000/courses'
-SEGMENT_KEY=null
+SEGMENT_KEY=''
 SITE_NAME='edX'
 SOCIAL_UTM_MILESTONE_CAMPAIGN='edxmilestone'
 STUDIO_BASE_URL='http://localhost:18010'
-- 
2.34.1

